src/lib/evil/evil_time.h

```diff
@@ -9,9 +9,11 @@
 #ifndef WIN32_LEAN_AND_MEAN
 # define WIN32_LEAN_AND_MEAN
 #endif
-#include <winsock2.h> // strcut timeval
+#include <evil_windows.h>
```

Comment:
This don't need to be inside `WIN32_LEAN_AND_MEAN`

Reply:
Now it doesn't need to be inside off `_MSC_VER`

Reply:
Yes, I will delete this last commit, just to test it, and to correct the previous push, as there were many conflicts because the base branch had not changed, now everything is fine.